### PR TITLE
US-28 Add inits/fulls on interviewers view and block editing them

### PR DESCRIPTION
**Pull request by kwidzinskidawid**

### CODE REVIEW

**Comment by RogowskaMalgorzata** [comment]
Seems good.

**Comment by r-kruk-kainos** [comment]
Code review done, everything's fine 👍

**Comment by dawidk-kainos** [comment]
looks good

File: src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js [hunk 1]
```diff
@@ -181,18 +181,24 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
...
+                    message: 'Something went wrong with sending your note.',
+                    delay: 2000});
             });
         }
 
         $scope.markSlots = function(slot) {
-            if($scope.relativeDayNumber < 0 && !$scope.isRecruiter){
-                return;
+            if(!$scope.isRecruiter) {
+                if(slot.type === AvailabilityEnum.full.id ||
```
Comment (rlewan) [inline src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js]: Complex conditional expressions obscure code readability. Consider extracting the content of this if into a function with a meaningful name instead.

**Pull request merged**